core/src/main/java/hudson/slaves/OfflineCause.java

```diff
@@ -212,5 +222,28 @@ public static class IdleOfflineCause extends SimpleOfflineCause {
         public IdleOfflineCause() {
             super(hudson.slaves.Messages._RetentionStrategy_Demand_OfflineIdle());
```

Comment:
If this wasn't shown before and now is I'd suggest changing the text from `computer` to `agent`

`Offline because computer was idle` doesn't make sense as a computer isn't user facing terminology, (not mentioned in https://www.jenkins.io/doc/book/glossary/), it should be agent instead.

Reply:
This was already shown before, just not before the agent was online, then became offline again as instructed by the retention strategy.